Source/Scene/Model/Model.js

```diff
@@ -355,6 +356,18 @@ function Model(options) {
   this._enableShowOutline = defaultValue(options.enableShowOutline, true);
   this._debugWireframe = defaultValue(options.debugWireframe, false);
 
+  // Warning for improper setup of debug wireframe
+  if (
+    this._debugWireframe === true &&
+    this._enableDebugWireframe === false &&
+    this.type === Model.GLTF
+  ) {
+    oneTimeWarning(
+      "model-debug-wireframe-ignored",
+      "enableDebugWireframe must be set to true in Model.Gltf, otherwise debugWireframe will be ignored."
```

Comment:
The method is named `fromGltf()`
```suggestion
      "enableDebugWireframe must be set to true in Model.fromGltf, otherwise debugWireframe will be ignored."
```